tools/mob_suite/mob_recon.xml

```diff
@@ -12,78 +13,133 @@
+
+    mob_recon  --num_threads \${GALAXY_SLOTS:-4}  --infile '${named_input}' --run_typer
+
     #if str($adv_param.unicycler_contigs) == "True":
       --unicycler_contigs 
     #end if   
-    #if str($adv_param.run_circlator) == "True":
-      --run_circlator 
-    #end if 
-    #if str($adv_param.min_length_condition.min_length_param) == "True":
-      --min_length '${adv_param.min_length_condition.min_length_value}'
+
+    #if str($adv_param.run_overhang) == "True":
+      --run_overhang
     #end if 
-    --run_typer --min_rep_evalue '${adv_param.min_rep_evalue}'
+    
+    #if $adv_param.debug:
+      --debug
+    #end if
+
+    #if $adv_param.plasmid_db
+   --plasmid_db $adv_param.plasmid_db
+   #end if
+
+   #if $adv_param.plasmid_mash_db
+   --plasmid_mash_db '$adv_param.plasmid_mash_db'
+   #end if
+
+   #if $adv_param.plasmid_meta
+   --plasmid_meta '$adv_param.plasmid_meta'
+   #end if
+
+   #if $adv_param.repetitive_mask
+   --repetitive_mask '$adv_param.repetitive_mask'
+   #end if
+
+   #if $adv_param.plasmid_mob
+   --plasmid_mob $adv_param.plasmid_mob
+   #end if
+   
+   #if $adv_param.plasmid_mpf
+   --plasmid_mpf '$adv_param.plasmid_mpf'
+   #end if
+
+   #if $adv_param.plasmid_orit
+   --plasmid_orit '$adv_param.plasmid_orit'
+   #end if
+
+    --min_length '${adv_param.min_length}'  
+    --min_rep_evalue '${adv_param.min_rep_evalue}'
     --min_rep_evalue '${adv_param.min_rep_evalue}'
     --min_mob_evalue '${adv_param.min_mob_evalue}'
     --min_con_evalue '${adv_param.min_con_evalue}'
     --min_rep_ident '${adv_param.min_rep_ident}'
     --min_mob_ident '${adv_param.min_mob_ident}'
     --min_con_ident  '${adv_param.min_con_ident}'
     --min_rpp_ident   '${adv_param.min_rpp_ident}'
+
+    --min_rep_cov '${adv_param.min_rep_cov}'
+    --min_mob_cov '${adv_param.min_mob_cov}'
+    --min_con_cov '${adv_param.min_con_cov}'
+    --min_rpp_cov '${adv_param.min_rpp_cov}'
     --outdir 'outdir' &&
     mkdir ./outdir/plasmids && (mv outdir/plasmid*.fasta ./outdir/plasmids 2> /dev/null || true)
   ]]>  
   </command>
   <inputs>
     <param name="input" type="data" format="fasta" label="Input" help="FASTA file with contig(s)"/>
     <section name="adv_param" title="Advanced parameters" expanded="False">
-      <param name="unicycler_contigs" label="Check for circularity flag generated by unicycler in contigs fasta headers" type="select" value="True">
+      <param name="unicycler_contigs" label="Check for circularity flag generated by unicycler in contigs fasta headers?" type="select" value="True">
         <option value="True">Yes</option>
         <option value="False">No</option>
       </param>
-      <param name="run_circlator" label="Run circlator minums2 pipeline to check for circular contigs" type="select" value="True">
+      <param name="run_overhang" label="Detect circular contigs (i.e. potential plasmids) with assembly overhangs?" type="select" value="True">
```

Comment:
Same with this. Couldn't you make this a boolean value as well?
Then at line 19, and 23, you can simply do:
```
#if $my_bool_var
...
```
Instead of comparing a string to True.

Reply:
Yes sure. I like this idea.